packages/web/src/services/audius-backend/BuyAudio.ts

```diff
@@ -98,41 +91,33 @@ export const pollForAudioBalanceChange = async ({
   maxRetryCount = DEFAULT_MAX_RETRY_COUNT
 }: {
   tokenAccount: PublicKey
-  initialBalance?: u64
+  initialBalance: bigint
```

Comment:
Is my understanding correct that an uninitialized account will have a balance of 0 now?

Reply:
I don't think so - rent is required for all token accounts. If the token account is unitialized `getAccount` will throw an error.
[We fallback `initialBalance` to 0 here.](https://github.com/AudiusProject/audius-protocol/blob/d8b07df28390ac76f9f452b906cf5ebe3078c59d/packages/web/src/store/application/ui/buy-audio/sagas.ts#L687) Would be nice to be able to accept `initialBalance` as `undefined` but didn't seem necessary.

Reply:
ah got it, thanks for the pointer!